### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by michaelhaxhiu**
- Added a space because it's missing & I have ocd.
- Added a few lines that basically clarify "implement" = deploying to production. If this feels super obvious to you as an engineer, then feel free to nix it.

This is a tiny change.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ New contributors are limited to working on one job at a time, however experience
...
+In this scenario, it’s possible that you found a bug or enhancement that we haven’t posted to the [Github repository](https://github.com/Expensify/App/issues?q=is%3Aissue). This is an opportunity to propose a job, and (optionally) a solution. If it's a valid job proposal that we choose to implement by deploying it to production — either internally or via an external contributor — then we will compensate you $250 for identifying and proposing the improvement. Note: If you get assigned the job you proposed **and** you complete the job, this $250 for identifying the improvement is *in addition to* the reward you will be paid for completing the job.
```
Comment (sketchydroide) [inline CONTRIBUTING.md:44]: was this change wanted as well?

Comment (sketchydroide) [reply CONTRIBUTING.md:44]: ok just reread the PR and it seems it was

**Comment by sketchydroide** [comment]
`Added a few lines that basically clarify "implement" = deploying to production. If this feels super obvious to you as an engineer, then feel free to nix it.` I think this is ok, a clarification of what implementing is

**Review by sketchydroide — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (michaelhaxhiu) [reply CONTRIBUTING.md:44]: yep!

**Pull request merged**